### DISCUSSION TITLE
Where to find reference from docstring documentation ?

### DISCUSSION THREAD

**Discussion by LuizFrra** [sqlalchemy/sqlalchemy#10932] [Usage Questions]
I'm trying to understand some parameters from the method create_engine. for example, the docstring has the following comment: 
```
    .. seealso::

        :doc:`/core/engines`

        :doc:`/dialects/index`

        :ref:`connections_toplevel`
```

But I don't know where I can find those reference or documentation mentioned in the comment above. I already tried through the sqlachemy website using:

* https://docs.sqlalchemy.org/en/20/core/engines
* https://docs.sqlalchemy.org/en/20/dialects/index

And also using the search tool:

https://docs.sqlalchemy.org/en/20/search.html?q=connections_toplevel&check_keywords=yes&area=default

But I could not found any reference or documentation.

![image](https://github.com/sqlalchemy/sqlalchemy/assets/20446456/916e012f-acb7-4723-9693-f5ee4e283913)
![image](https://github.com/sqlalchemy/sqlalchemy/assets/20446456/33be5c54-f987-45f3-a6e7-1137c94ec16a)

**Comment by CaselIT** [comment]
Hi,

Those are rst reference marking. The best option is to use consul the documentation of the section where they are specified. For example in create engine you would search for create engine go to https://docs.sqlalchemy.org/en/20/core/engines.html#sqlalchemy.create_engine and look at the see also box
